main.cpp

```diff
@@ -212,17 +243,44 @@ struct Range
         int controlPanelWidth = 24;
         bool supportsWifi = true;
         bool isAnalog = false;
+        int maxKnobTurned = 10;
 
         void printDaysLeft(int warrantyBy);
-        void informCurrentTime (bool timeUpdated);
-        void indicateRepairDate (std::string date, bool needsRepair);
-        bool selfCleans (bool heavyCleaning = true, std::string setting = "");
-        
+        void informCurrentTime(bool timeUpdated);
+        void indicateRepairDate(std::string date, bool needsRepair);
+        bool selfCleans(bool heavyCleaning = true, std::string setting = "");
+        void turnUpTemp(int ovenTemp = 350)
+        {
+            while(ovenTemp <= 355)
+            {
+                int degreeKnobTurned;
+                for(degreeKnobTurned = 0; degreeKnobTurned <= maxKnobTurned; ++degreeKnobTurned)
+                {
+                    ovenTemp += 1;
+                    std::cout << "Increasing the oven temperature." << std::endl;   
+                }
+            break;
+            }
```

Comment:
remove the `break`, because it prevents the `while` loop from looping.
- [ ] revise